osu.Game.Tournament/Components/RoundAnimation.cs

```diff
@@ -27,6 +27,7 @@ public partial class RoundAnimation : VisibilityContainer
         public const float DISC_SIZE = 400;
 
         private const float border_width = 5;
+        private static bool isAnimationRunning = false;
```

Comment:
Theoretically this would not work, since we always add a new instance of `RoundAnimation` by `new RoundAnimation()`, and all of them have their own `isAnimationRunning`, not interfering with each other.

Or this is to do something else?

Reply:
Just forget this. I must be struggling with shit when posting this comment.

![Pure Dolly shit](https://github.com/user-attachments/assets/882094d9-4b99-424e-93c4-ff98c4acaf47 "Pure Dolly shit")

Reply:
Since you hava managed to solve this issue I will mark this conversation as resolved.
Apoligies for any inconveniences.